tsconfig.json

```diff
@@ -3,15 +3,17 @@
 	"compilerOptions": {
 		"alwaysStrict": true,
 		"lib": ["esnext"],
+		"strict": true,
 		"module": "commonjs",
 		"noUnusedParameters": true,
 		"outDir": "./dist",
 		"sourceMap": true,
-		"types": ["node"],
-        "declaration": true,
-        "noUnusedLocals": true,
-        "removeComments": false,
-        "target": "esnext"
+		"types": ["node", "discord.js", "klasa"],
+		"declaration": true,
+		"declarationMap": true,
+		"noUnusedLocals": true,
+		"removeComments": false,
+		"target": "esnext"
 	},
 	"exclude": [
 		"node_modules",
```

Comment:
Entire exclude block can be removed. These dirs are not inside the include filter. From the docs: 

```
Files included using "include" can be filtered using the "exclude" property. However, files included explicitly using the "files" property are always included regardless of "exclude". The "exclude" property defaults to excluding the node_modules, bower_components, jspm_packages and <outDir> directories when not specified.
```